packages/filesystem/src/node/node-filesystem.spec.ts

```diff
@@ -51,8 +51,11 @@ describe("NodeFileSystem", function () {
             expect(fs.statSync(FileUri.fsPath(uri)).isFile()).to.be.true;
 
             const stat = await fileSystem.getFileStat(uri.toString());
-            expect(stat.isDirectory).to.be.false;
-            expect(stat.uri).to.eq(uri.toString());
+            expect(stat).to.not.be.undefined;
+            if (stat) {
+                expect(stat.isDirectory).to.be.false;
```

Comment:
I would go with `stat!.isDirectory`. No need to check if it is `undefined` or not. It is our test, should fail in line 54 anyway.

Reply:
Ah yes `stat!` makes sense for the tests. Thanks!